handbook/direction/code-graph/index.md

```diff
@@ -0,0 +1,41 @@
+# Code Graph
+
+The Code Graph organization comprises teams working on search, insights, code intelligence, and batch changes, and this page describes the overarching mission and strategy that ties them together. Each one also has an individual direction page, indexed from our [direction home page](../index.md#code-graph), that describes what they are working on and why in more detail.
+
+## Mission
+
+We build the Code Graph to make working with code easier, regardless of how much you have, how complex it is, where you store it, or even how technical you are.
+
+## Vision
+
+### One year vision
+
+> Within the next year, we want to ensure that we are solving the most critical problems for our enterprise users, focusing on retaining existing and acquiring new enterprise customers. By providing as much value as possible to them, we can grow the business while solving the hardest scaling and complexity problems for everyone.
+
+### Three year vision
+
+> Within three years, we imagine a much more connected Code Graph experience, not just in terms of the kinds of searching that you're able to do, but in how we can offer suggestions for interesting searches and what you can do with the results. Expanding on capabilities we've introduced recently, such as semantic search, insights, and batch changes, we will offer an unparalleled solution for searching and making decisions across branches, repositories, and even code hosts, beyond even what's possible today.
+
+### Ten year vision
+
+> At the ten year mark, we want to make using Sourcegraph to understand and modify code so easy that anyone can do it, including people not traditionally considered developers. We want to [democratize coding](../../company/strategy.md##10-year-vision) by making it so that everyone in the company can engage with the organization's code, safely making changes to implement their department's priorities and interests, without need for an intermediary to do it for them.
```

Comment:
I think there's an extra `#` in that link

Reply:
Good eye, thanks!

```suggestion
> At the ten year mark, we want to make using Sourcegraph to understand and modify code so easy that anyone can do it, including people not traditionally considered developers. We want to [democratize coding](../../company/strategy.md#10-year-vision) by making it so that everyone in the company can engage with the organization's code, safely making changes to implement their department's priorities and interests, without need for an intermediary to do it for them.
```